### ISSUE TITLE
[Bug] sha256sums use hardcoded location

### ISSUE THREAD

**Issue by Lerentis** [awslabs/soci-snapshotter#1010]
### Description

Hello there, 

i noticed that the sha256sum files use a hardcoded path of the archive which makes a validation fail if the path there is not the same, which i assume will be the case for most people. it would be great to just have a relative path here as it was in v0.4.0

### Steps to reproduce the bug

```bash
_version=v0.4.1
release_archive="soci-snapshotter-${_version:1}-linux-amd64-static.tar.gz"
release_shasums="${release_archive}.sha256sum"
cd /tmp
curl -sL "https://github.com/awslabs/soci-snapshotter/releases/download/v0.4.1/${release_archive}" -o "/tmp/${release_archive}"
curl -sL "https://github.com/awslabs/soci-snapshotter/releases/download/v0.4.1/${release_shasums}" -o "/tmp/${release_shasums}"
echo "${release_shasums}"
cat "${release_shasums}"
sha256sum -c "${release_shasums}"; 
```

### Describe the results you expected

soci-snapshotter-0.4.1-linux-amd64-static.tar.gz.sha256sum to contain this:
```
00474043ffed48a70ef88e80737722cf39c029a9744cab1071edfbebf403e93e  soci-snapshotter-0.4.1-linux-amd64-static.tar.gz
```

instead of this: 
```
00474043ffed48a70ef88e80737722cf39c029a9744cab1071edfbebf403e93e  /home/runner/work/soci-snapshotter/soci-snapshotter/release/soci-snapshotter-0.4.1-linux-amd64-static.tar.gz
```

### Host information

1. OS: Arch Linux
2. Snapshotter Version: v0.4.1
3. Containerd Version:  not relevant

**Comment by Lerentis** [comment]
Duplicate of #1007

**Comment by Lerentis** [comment]
or not. just validated and the PR to fix #1007 only fixes the archive but not the shasums generation

**Comment by austinvazquez** [comment]
@Lerentis, the artifacts and shasums for v0.4.1 release have been regenerated to resolve the issue. Thanks for reporting.
